### ISSUE TITLE
Ionide-fsharp did not support FSharp.Data well, the fsharp code is correct and able run when I press atl+enter, but ionide show error hint in IDE

### ISSUE THREAD

**Issue by ScottHuangZL** [ionide/ionide-vscode-fsharp#1437]
**Describe the bug**
Below code is running well when I press alt-enter. However IDE show error hint at  `fsi` and `CsvProvider` point. 
**I guess the ionide did not figure out the correct lib reference of FSharp.Data and report the wrong alerts due to refer to FSharp.Core.**
I try to uninstall Ionide current 4.7x version and install 5.0.0-preview1, same error.

I try many times, and the wear things is there happen have one time the ide is ok, not show error and can show intelligent code hint of `CsvProvider`   
After I close vscode and reopen, IDE show error again.    

**To Reproduce**
Steps to reproduce the behaviour:
Full code as below, which is simple:
`#load ".paket/load/netcoreapp3.1/main.group.fsx"
open System
open FSharp.Data
fsi.AddPrinter<DateTime>(fun d -> d.ToShortDateString())
[<Literal>]
let SampleDataFile =
    __SOURCE_DIRECTORY__
    + "./csse_covid_19_data/csse_covid_19_daily_reports/01-22-2020.csv"
type Covid = CsvProvider<SampleDataFile>
Covid.GetSample().Rows
let files =
    System.IO.Directory.GetFiles("./csse_covid_19_data/csse_covid_19_daily_reports")
    |> Seq.map System.IO.Path.GetFullPath
let allData =
    files
    |> Seq.collect
        (fun r ->
            let data = Covid.Load r
            data.Rows)
`

**Expected behaviour**
Ionide should not report error hint.  Ionide should need provide code inteligent of `CsvProvider` 

**Screenshots**

**Environment (please complete the following information):**
 - OS: WIN10
 - Ionide version: 5.0.0-preview (I try 4.7 version at very beginning, also not work)
 - VSCode version: 1.51.1
 - dotnet SDK version: .Net 5
 - mono / .Net Framework version:

**Additional context**

**Comment by ScottHuangZL** [comment]
The key IDE error messages as below:

**The value, namespace, type or module 'fsi' is not defined. Maybe you want one of the following:
   fst**

**The type 'CsvProvider' is not defined.**

Actually the code is correct and working well if I press Alt+Enter, so, it is Ionide error.

**Comment by ScottHuangZL** [comment]
I find a way to make Ionide did not show errors with below steps:

1. Ensure the *.fsx closed in vscode firstly (such as press ctrl+w to close it).  This step is mandatory.
2. I try to delete .ionide and .paket foldres, and then paket restore, and then paket generate load scripts
3. Wait VSCode IDE show "Activating Extentions" at bottom left, wait it complete.
4. And then you can open any *.fsx, it will work well.

I try to close VSCode and try several times, and find step #2 can ignore.

**So in summary, the solution steps as below:**
1. Ensure close *.fsx firstly    
2. Wait VSCode activating extensions
3. Then you can open *.fsx now.

Can IonIde enhanced to ignore close *.fsx firstly or ensure VSCode complete activating extensions firstly? Or else, it is inconvenient and confuse people.

**Comment by backerman** [comment]
Confirmed with Ionide 5.0-preview2, VS Code 1.51.1, Win10 20H2. I don't have a .paket folder to remove because I was trying a script with `"#r nuget: FSharp.Data"`, but I did remove $Env:HOMEPATH\.nuget and .ionide in the opened folder; that workaround was insufficient.

**Comment by nhirschey** [comment]
@ScottHuangZL , check with ionide 5.0.3 released today. This code below works with no intellisense errors for me on windows.

```fsharp
#r "nuget: FSharp.Data"

open System 
open FSharp.Data 
fsi.AddPrinter<DateTime>(fun d -> d.ToShortDateString()) 

[<Literal>] 
let SampleDataFile = __SOURCE_DIRECTORY__ + "/01-22-2020.csv" 
type Covid = CsvProvider<SampleDataFile>
Covid.GetSample().Rows 
```

**Comment by Krzysztof-Cieslak** [comment]
Everything seems to be working fine with latest 5.0, and `#r nuget`
